src/components/form/form-summary/form-summary.scss

```diff
@@ -33,3 +35,8 @@
     }
   }
 }
+
+.carbon-form-summary--invalid {
+  background-color: $grey-dark-blue-5;
+  margin-left: 15px;
```

Comment:
Is this internal to the Summary as a whole (I'm thinking it is)? It sort of feels like a similar problem if this extends outside of it's box!